discovery-provider/src/queries/queries.py

```diff
@@ -98,6 +98,12 @@ def get_users_route():
     current_user_id = get_current_user_id(required=False)
     args["current_user_id"] = current_user_id
     users = get_users(args)
+
+    def validate_hidden_fields(user, current_user_id):
+        if "playlist_library" in user and (not current_user_id or current_user_id != user["user_id"]):
+            del user["playlist_library"]
```

Comment:
this is prob dumb question but this only affects the returned value of the api and shouldn't actually delete any data in the backend right?

Reply:
yup!